### DISCUSSION TITLE
[Docs Site] Auto scrolling down (scrolling so fast) when holding down left mouse button and dragging on code blocks

### DISCUSSION THREAD

**Discussion by razonyang** [twbs/bootstrap#39633] [General] [answered]
![getbootstrap](https://github.com/twbs/bootstrap/assets/17720932/a97aeafc-9f26-4f67-a358-df74b148a46b)

Browsers: Chrome 121.0.6167.86, Edge 121.0.2277.83 
OS: Windows 11

**Comment by julien-deramond** [comment, marked as answer]
Thanks for reporting this issue @razonyang 
It has been already identified here: https://github.com/twbs/bootstrap/issues/39605
